server/src/main/java/org/opensearch/action/admin/cluster/decommission/awareness/delete/DeleteDecommissionAction.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.cluster.decommission.awareness.delete;
+
+import org.opensearch.action.ActionType;
+
+/**
+ * Transport action delete decommission.
+ *
+ * @opensearch.internal
+ */
+
+public class DeleteDecommissionAction extends ActionType<DeleteDecommissionResponse> {
```

Comment:
As part of one comment of this PR - https://github.com/opensearch-project/OpenSearch/pull/4261, we decided to rename it to DeleteDecommissionStateAction

Reply:
Done.